config/core/resources/channel.yaml

```diff
@@ -249,11 +265,3 @@ spec:
     shortNames:
       - ch
   scope: Namespaced
-  conversion:
-    strategy: Webhook
-    webhook:
-      conversionReviewVersions: ["v1", "v1beta1"]
```

Comment:
do we want to remove this, with others?  

Looking at serving, when they removed if for, while (completely) "dropping" serving `v1beta1`, like: https://github.com/knative/serving/pull/9620/files

Reply:
I am going through each resource and updating the schema and validating webhooks.